From 00a5eba6881174a06fcdab69a76722c9e633b710 Mon Sep 17 00:00:00 2001
From: Steve Ellcey <sellcey@caviumnetworks.com>
Date: Thu, 31 Aug 2017 17:46:27 +0100
Subject: [PATCH 03/10] Add ILP32 support to aarch64 in build-many-glibcs.py

2017-09-01  Steve Ellcey  <sellcey@caviumnetworks.com>

	* scripts/build-many-glibcs.py (Context.add_all_configs):
	Add ilp32 variants of aarch64 architecture.
---
 scripts/build-many-glibcs.py | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/scripts/build-many-glibcs.py b/scripts/build-many-glibcs.py
index 5be9b79019..d8c2c7c34e 100755
--- a/scripts/build-many-glibcs.py
+++ b/scripts/build-many-glibcs.py
@@ -163,8 +163,16 @@ class Context(object):
                         os_name='linux-gnu',
                         extra_glibcs=[{'variant': 'disable-multi-arch',
                                        'cfg': ['--disable-multi-arch']}])
+        self.add_config(arch='aarch64',
+                        os_name='linux-gnu',
+                        variant='ilp32',
+                        gcc_cfg=['--with-multilib-list=ilp32'])
         self.add_config(arch='aarch64_be',
                         os_name='linux-gnu')
+        self.add_config(arch='aarch64_be',
+                        os_name='linux-gnu',
+                        variant='ilp32',
+                        gcc_cfg=['--with-multilib-list=ilp32'])
         self.add_config(arch='alpha',
                         os_name='linux-gnu')
         self.add_config(arch='arm',
-- 
2.11.1

